picopass/picopass_device.h

```diff
@@ -110,6 +117,7 @@ typedef struct {
     uint8_t key_block_num; // in loclass mode used to store csn#
     bool loclass_mode;
     bool loclass_got_std_key;
+    uint8_t loclass_mac_buffer[8 * LOCLASS_NUM_PER_CSN];
```

Comment:
is this 8 because that is the mac length?

Reply:
4 bytes for NR plus 4 bytes for MAC, yeah